lib/Parse/ParseType.cpp

```diff
@@ -184,11 +184,11 @@ ParserResult<TypeRepr> Parser::parseTypeSimple(Diag<> MessageID,
     TypeSpecifier = VarDecl::Specifier::InOut;
   } else if (Tok.is(tok::identifier)) {
     if (Tok.getRawText().equals("__shared")) {
-      assert(false);
+      assert((false));
```

Comment:
This and all the others should be llvm_unreachable

Reply:
Yes, true. Blindly trying to kill off warnings without paying enough attention. Thanks for noticing.

Reply:
Ah sorry I removed this part in https://github.com/apple/swift/pull/13173